pandas/tests/frame/test_repr_info.py

```diff
@@ -140,9 +140,12 @@ def test_unicode_string_with_unicode(self):
         df = DataFrame({'A': ["\u05d0"]})
         str(df)
 
-    def test_bytestring_with_unicode(self):
-        df = DataFrame({'A': ["\u05d0"]})
-        bytes(df)
+    def test_str_to_bytes_raises(self):
```

Comment:
I think OK to just remove these tests; error message just matches stdlib

Reply:
Why not just keep them? these things are now (after this PR) supposed to raise,so just have a test for that?

Reply:
I'm not hugely tied to it but these tests are making assertions about things that the stdlib does instead of something pandas explicitly does, so just seems strange to me that have that in our test suite. Let's see what others think

Reply:
Ok, I see your point. Not a hard point for me either. I can remove them later tonight.

Reply:
i think these tests are fine; pretty innocuous and prevent adding back a ``__bytes__``.